.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: "🐛 Bug Report"
+description: Create a new ticket for a bug.
+title: "🐛 [BUG] - <title>"
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an explicit description of your issue
+      placeholder: Short and explicit description of your incident...
+    validations:
+      required: true
+  - type: textarea
+    id: reprod
+    attributes:
+      label: "Reproduction steps"
+      description: Please enter an explicit description of your issue
+      value: |
+        1. Go to '...'
```

Comment:
Since this is a library rather than an application, it might be better to request a minimal code example demonstrating the problem. Requesting explicitly 'what output did you get?' and 'what output did you expect?' is probably also a good idea.

Reply:
Good suggestion. I've tweaked a new version but don't believe there is a way of testing until its merged in?